components/playlist/browser/resources/components/app.v1.tsx

```diff
@@ -22,22 +28,34 @@ const HeaderWrapper = styled.header<{ isPlaylistPlayerPage: boolean }>`
 `
 
 export default function App () {
+  const lastPlayerState = useLastPlayerState()
+  const editMode = usePlaylistEditMode()
+
   return (
     <>
       <Route
         path={'/playlist/:playlistId'}
         children={({ match }) => {
           const playlistId = match?.params.playlistId
           return (
-            <HeaderWrapper isPlaylistPlayerPage={!!playlistId}>
-              <Header playlistId={playlistId} />
-            </HeaderWrapper>
+            <>
+              <HeaderWrapper isPlaylistPlayerPage={!!playlistId}>
+                <Header playlistId={playlistId} />
+              </HeaderWrapper>
+              <VideoFrame
+                visible={
+                  !!lastPlayerState?.currentItem &&
```

Comment:
Optional: The only truthy/falsey values you need to be careful with are `numbers`, as React will render `0` instead of nothing. All other falsey values (`false`, `undefined`, `NaN`, `''`, `null`) won't be rendered, so it's safe to leave off the `!!` here, if you want :smile:

Reply:
Thanks, good to know that! Here `!!` seems to be needed unless we change the prop's type to `bool | undefined`. But I feel like `bool | undefined` is a little bit awkward.

Reply:
I often use `visible?: boolean`, where the absence of the boolean is falsey, but this is 100% fine as is :smile: